src/backend/services/wallet-api/src/routes.py

```diff
@@ -31,6 +36,39 @@ def getWalletAmount():
     response = send_get_request( blockchain_wallet_url.format("balance"), data)
     return jsonify(response.json()), response.status_code
 
+@cross_origin()
+@app.route("/history", methods=['GET'])
```

Comment:
use `/history/<umnetId>` instead of using args.

Reply:
+1, the front end is sending the umnetId as a query parameter, not a payload!